lib/pbench/test/unit/server/test_unpack_tarballs.py

```diff
@@ -184,18 +196,16 @@ def mock_symlink(path: Path, dest: Path):
 
         linkedlistdest = "TO-INDEX"
```

Comment:
[As I mentioned before](https://github.com/distributed-system-analysis/pbench/pull/2975#discussion_r983838307), I think this would be better as
```
        linkedlistdest = MockConfig.get("pbench-server", "unpacked-states").split(", ")[0]
```
or something similar.